pkg/auth/user/util.go

```diff
@@ -25,3 +27,14 @@ func ConvertAttributes(attrMap map[string][]string) []*v1.UserAttribute {
 	})
 	return result
 }
+
+// LogSuccessfulUserLogin logs user attributes in the specified logger instance.
+func LogSuccessfulUserLogin(log *logging.Logger, user *v1.AuthStatus) {
```

Comment:
Its a bit awkward to have the logger being passed here. Does it really matter that the origin package will be the auth service?

Reply:
I think the reason why it matters is because we call this function from two places, so having those calls distinguished could be helpful in debugging

Reply:
I think this could be separated into a function that just returns the logging context as an array of `zap.Field`. Then the caller calls the logger, and we just pass the logging context.

Reply:
We agreed with @stehessel that having a special function to call `log.Warnw` makes sense because it allows guaranteeing several callers will log users with the same format, message and log level.
However, it still is beneficial to extract the function that creates logging context - `extractUserLogFields` as it allows for easier unit testing.